api/controllers/v1/jobs.ts

```diff
@@ -187,6 +204,11 @@ async function NotifyBuildSummary(jobId: string): Promise<any> {
       const prCommentId = await githubCommenter.getPullRequestCommentId(fullDocument.payload, pr);
       const fullJobDashboardUrl = c.get<string>('dashboardUrl') + jobId;
 
+      // We currently avoid posting the Gatsby Cloud preview url on GitHub to avoid
+      // potentially conflicting behavior with the S3 staging link with parallel
+      // frontend builds. This is in case the GC build finishing first causes the
+      // initial comment to be made with a nullish S3 url, while subsequent comment
+      // updates only append the list of build logs.
```

Comment:
This is something we should probably handle separately to test its behavior, and to avoid breaking the feature as part of this PR